content/02.main-text.md

```diff
@@ -1,3 +1,21 @@
 ## Main text {.page_break_before}
 
-*Work in progress*
+### Introduction
+
+Open research – which includes sharing code, data, and manuscripts – benefits the researchers who engage in open practices [@doi:10.7554/eLife.16800], their scientific peers, and the public.
+`TODO: more references needed`
+Here we advocate for the benefits of writing review articles openly, specifically planning, organizing, writing, and editing this type of manuscript collaboratively in an unrestricted public forum.
+Reviews presenting the state of the art in a scientific field are often prepared by a single research group or a small team of colleagues.
+In contrast, broadly opening the writing process to anyone engaged in the topic can help maximize the review's value by ensuring that it represents diverse opinions and does not neglect relevant research.
+Review authors can engage with the authors of original research to clarify their methods and results and present them accurately.
+In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review `TODO: define this or provide a reference?`, incentivizing it with potential manuscript authorship.
+However, inviting wide authorship brings many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
+
+We present potential solutions to these challenges based on our recent experience leading a collaborative review "Opportunities And Obstacles For Deep Learning In Biology And Medicine" [@doi:10.1101/142760].
+`TODO: and the SciHub manuscript, or add that at the end to show how these practices generalize?`
```

Comment:
Think scihub should come later - maybe discussion/conclusion - since this is primarily about review articles. I also see the benefits for work like that, but I think the strongest case is reviews where fair coverage of the literature is critical to success.